src/mrpro/data/KHeader.py

```diff
@@ -251,6 +253,8 @@ def from_ismrmrd(
 
         # Dump everything into misc
         parameters['misc'] = dataclasses.asdict(header)
+        # Remember encoding number
```

Comment:
Maybe the comment could explain why, instead of what...? (I don't know why we save it)